Upstream: taken from https://github.com/aio-libs/aiohttp/commit/ce50b036e75852b57a99389f2735f8c2153efb14
Reason: We use py3-yarl>1.6.0 (1.6.2 at the time of writing this), so relax deps

diff --git a/setup.py b/setup.py
index 9913039..8039b51 100644
--- a/setup.py
+++ b/setup.py
@@ -76,7 +76,7 @@ install_requires = [
     'chardet>=2.0,<4.0',
     'multidict>=4.5,<5.0',
     'async_timeout>=3.0,<4.0',
-    'yarl>=1.0,<1.6.0',
+    'yarl>=1.0,<2.0',
     'idna-ssl>=1.0; python_version<"3.7"',
     'typing_extensions>=3.6.5; python_version<"3.7"',
 ]
